### ISSUE TITLE
tkmktap is broken and probably useless, eliminate it

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2020]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#2020|
|Type|task|
|Created|2006-08-27 17:37:13Z|

<details><summary>Searchable metadata</summary>

```
trac-id__2020 2020
type__task task
reporter__exarkun exarkun
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1156700233000000 1156700233000000
changetime__1159050687000000 1159050687000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_1"></a>|@radix removed owner|
|-|-|

Please review.

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)<a name="note_2"></a>|@jml set owner to @radix|
|-|-|

Note: code in remove-tkmktap-2020

This looks good.  I can't personally test the changes to the win32 install code, so I'm assuming you've got it right. 

Also, you appear to have introduced some doc changes for web. It would be better if they were in another branch, but it's ok to merge this branch as it is. In any case, please remember to close the tickets that match those doc changes (in the range of [#2025](https://github.com/twisted/twisted/issues/2025) to [#2030](https://github.com/twisted/twisted/issues/2030)).

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_3"></a>|@radix commented|
|-|-|

Thanks. The changes in that branch to twisted/web were unintentionally committed. I'll merge the stuff other than that.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_4"></a>|@radix set status to `closed`|
|-|-|

(In [18271]) Merge remove-tkmktap-2020

Author: radix
Reviewer: jml
Fixes [#2020](https://github.com/twisted/twisted/issues/2020)

Gets rid of a broken thing that was floating around in our repository
(tkmktap). Also updates all documentation and distribution stuff that 
refers to it.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
